lib/blacklight/marc/catalog.rb

```diff
@@ -12,7 +12,10 @@ def librarian_view
       @response, deprecated_document = search_service.fetch params[:id]
       @document = ActiveSupport::Deprecation::DeprecatedObjectProxy.new(deprecated_document, "The @document instance variable is deprecated and will be removed in Blacklight-marc 8.0")
       respond_to do |format|
-        format.html
+        format.html do
+          return render layout: false if request.xhr?
+          # Otherwise draw the full page
+        end 
         format.js { render :layout => false }
```

Comment:
Can we remove the `js` format now? I think that got removed from Blacklight, no?

Reply:
Yes, there's no need for that.